webpack.config.js

```diff
@@ -45,7 +45,7 @@ const config = {
 							loader: 'sass-loader',
 							query: {
 								includePaths: [ 'editor/assets/stylesheets' ],
-								data: '@import "variables";',
+								data: '@import "variables"; @import "mixins";',
```

Comment:
Hmm, I swear when I'd last looked one had the underscore and the other not, but it appears correct now, yes.